pkg/chain/local/local_test.go

```diff
@@ -63,7 +63,7 @@ func TestOnSignatureRequested(t *testing.T) {
 	keepAddress := common.Address([20]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1})
 	digest := [32]byte{1}
 
-	keep := localChain.OpenKeep(keepAddress, []common.Address{})
+	keep := localChain.OpenKeep(keepAddress, keepAddress, []common.Address{})
```

Comment:
Adding in the concept that local keeps have owning addresses means that we have to supply owners in all of the tests that don't actually care. I defaulted to making them be owned by the keep ID, but I'm up for more elegant solutions

Reply:
Dang that's a great solve to avoid the event hop. 

I would go with the empty address tbh.